docs/installation.md

```diff
@@ -13,11 +13,11 @@
 Update `./hack/config.yaml`:
```

Comment:
It's unrelated to this PR, but don't you also need to install kubeflow in `Dockerized kernel` requirements?